examples/reliable_conversation/src/tasks/task_functions.py

```
@@ -0,0 +1,900 @@
+import json
+import uuid
+from typing import Dict, Any, List
+from mcp_agent.agents.agent import Agent
+
+# Import our utilities
+import sys
+from pathlib import Path
+
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from utils.config import get_llm_class
```

Pylint (3.3.7)

[error] Unable to import 'utils.config'

(E0401)